content/docs/command-reference/commit.md

```diff
@@ -140,83 +140,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the parameters for that script. The parameters are defined in
+the `params.yaml` file. Updating the value of the `max_features` param to 6000
+changes the resulting model:
 
-```python
-bag_of_words = CountVectorizer(stop_words='english',
-            max_features=6000, ngram_range=(1, 2))
+```yaml
+featurize:
+  max_features: 6000
+  ngrams: 2
 ```
 
-This edit introduces a change that would cause the `featurize.dvc`, `train.dvc`
-and `evaluate.dvc` stages to execute if we ran `dvc repro`. But if we want to
-try several values for `max_features` and save only the best result to the
-cache, we can run it like this:
+This edit introduces a change that would cause the `featurize` and `train`
+stages to execute if we ran `dvc repro`. But if we want to try several values
+for `max_features` and save only the best result to the cache, we can run it
+like this:
```

Comment:
Turns out the text was correct, per https://github.com/iterative/dvc.org/pull/1626#pullrequestreview-465506364. Please revert...

Except I would change "if we ran `dvc repro`" for "if we reproduced the pipeline" here.

Reply:
> Except I would change "if we ran dvc repro" for "if we reproduced the pipeline" here.

Never mind. It's useful to have that link to the repro cmd ref.